infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/OperationalPointDao.kt

```diff
@@ -232,8 +236,8 @@ class OperationalPointDao(
     private fun getOperationalPoint(rs: ResultSet): OperationalPoint =
         OperationalPoint(
             name = rs.getString("name").let(::OperationalPointName),
-            abbreviation = rs.getString("abbreviation")?.let(::OperationalPointAbbreviation),
-            uicCode = rs.getString("uic_code").let(::UicCode),
+            abbreviation = rs.getUnsafeStringOrNull("abbreviation")?.toString()?.let(::OperationalPointAbbreviation),
+            uicCode = rs.getUnsafeStringOrNull("uic_code")?.toString()?.let(::UicCode),
```

Comment:
Oletan että näissä `getUnsafeStringOrNull()`-käytöissä ideana on kenttien nullability koska `rs.getString()` hanskaa sen väärin, vai? En ole ihan varma mitä mieltä olen niiden käytöstä siihen, koska niiden ideana on että kentissä voi olla dataa mitä me ei olla sanitoitu. Nämä haut kohdistetaan layout-puolen tauluihin, joten kaiken pitäisi olla jo sanitoitua 🤔 Mielestäni meidän ehkä kannattaisi vaan tehdä jonkinlaiset omat `getString()`- ja `getStringOrNull()`-variantit hanskaamaan näitä tilanteita, joskin näille tietotyypeile voisi harkita myös ihan kokonaan omiakin hakufunktioita `ResultSet`:iin

Reply:
Joo tuo että tässä kohdassa on räjähdys mahdollista on välitila, kirjasin tiketiksi GVT-3345 että putsataan (tässä kun on jäljellä vielä pari muutakin tapausta minkä takia voi paukkua UicCodet tai muut). Siinä tulee varmasti miettimistä että mikä tämän prosessin rakenne halutaan tarkkaan ottaen olevan, mitä voidaan olettaa Ratkosta tulevasta datasta, mitä operaattori voi tarkistaa, jne., siksi siis omana tikettinään.